lib/ui/view/channel/add_people/channel_add_people_view.dart

```diff
@@ -17,6 +21,7 @@ class ChannelAddPeopleView extends StatelessWidget {
         disposeViewModel: false,
         builder: (context, viewModel, child) => Scaffold(
           appBar: AppBar(
+<<<<<<< Updated upstream
             title: Text(
               'Add People',
               style: GoogleFonts.lato(
```

Comment:
use the const textStyles, it can be found in the styles.dart in the shared folder